.github/workflows/ci.yml

```diff
@@ -81,6 +81,29 @@ jobs:
       with:
         tool: wasm-bindgen-cli
 
+    - name: Install nightly rust to generate `-Zminimal-versions` lockfile (for MSRV)
+      if: matrix.rust_version == '1.71.0'
+      uses: dtolnay/rust-toolchain@master
```

Comment:
Nit: Use `hecrj/setup-rust-action@v2` (or change both places to use the same).

Reply:
Or maybe eschew a GitHub Action-thing, and just do `rustup install nightly-...` in the bash script below